package.json

```diff
@@ -61,13 +61,14 @@
   },
   "scripts": {
     "build": "babel src -d dist --copy-files --source-maps inline",
+    "prestart": "flow-remove-types node_modules/mapbox-gl/js/**/*.js --out-dir . >/dev/null 2>&1",
```

Comment:
How about `npm test`? With this approach, won't we need a `pretest` as well (or we have to run `npm start` before `npm test`)?

Also, having `flow-remove-types` run every time we `npm start` seems unnecessary. We really only need to run it once after an install.

How about adding it to `npm build`?
```
"build": "npm run flow-remove-types && babel src -d dist --source-maps inline",
"flow-remove-types": "flow-remove-types node_modules/mapbox-gl/js/**/*.js --out-dir . >/dev/null 2>&1",
```